src/test/java/seedu/address/storage/JsonAdaptedTransactionTest.java

```diff
@@ -25,14 +31,18 @@ public class JsonAdaptedTransactionTest {
 
     private static final String VALID_NAME = SHOPPING.getName().toString();
     private static final double VALID_AMOUNT = SHOPPING.getAmount().amount;
-    private static final String VALID_CATEGORY = SHOPPING.getCategory().category;
     private static final String VALID_DATETIME = SHOPPING.getDateTime().originalString();
     private static final String VALID_LOCATION = SHOPPING.getLocation().location;
     private static final String VALID_TYPE = SHOPPING.getType().toString();
+    private static final List<JsonAdaptedCategory> VALID_CATEGORIES = SHOPPING.getCategories().stream()
+            .map(JsonAdaptedCategory::new)
+            .collect(Collectors.toList());
 
     @Test
     public void toModelType_validPersonDetails_returnsPerson() throws Exception {
+        System.out.println(SHOPPING);
         var transaction = new JsonAdaptedTransaction(SHOPPING);
+        System.out.println(transaction);
```

Comment:
Any reason to leave these logging statements in?

Reply:
Oops, missed them when I was clearing my SOUT